src/types.ts

```diff
@@ -0,0 +1,6 @@
+type Todo = {
```

Comment:
"todo" is not an object anymore. It's an array.

Reply:
Done